packages/evm/src/stack.ts

```diff
@@ -1,5 +1,10 @@
 import { ERROR, EvmError } from './exceptions.js'
 
+type PeekCache = {
+  num: number
```

Comment:
This `num` should be the same as `peekArray.length` - right?

Reply:
Yes, maybe both wrongly named and not necessary on second thought. 😂